cart-service/src/lib/dovetech-response-mapper.test.ts

```diff
@@ -500,7 +500,7 @@ it('no actions should be returned if type is Order', () => {
 
   expect(result).toEqual({
     success: true,
-    actions: [],
+    actions: [buildSetCustomTypeAction(dtResponse, currencyCode, '[]')],
```

Comment:
Could you rename this test please as there is an action returned now if the type is order.

Reply:
renamed